Doc/library/stdtypes.rst

```diff
@@ -1131,6 +1131,9 @@ Lists are mutable sequences, typically used to store collections of
 homogeneous items (where the precise degree of similarity will vary by
 application).
 
+Despite its name, lists are more akin to arrays in other languages than to
+linked lists, since its elements are accessed with time complexity O(1).
```

Comment:
Use plural: “_their_ name”, “_their_ elements”, to agree with “lists are”

Reply:
:sweat_smile: thanks!